other-cel/restrict-deprecated-registry/restrict-deprecated-registry.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-deprecated-registry
+  annotations:
+    policies.kyverno.io/title: Restrict Deprecated Registry in CEL expressions
+    policies.kyverno.io/category: Best Practices, EKS Best Practices in CEL 
+    policies.kyverno.io/severity: high
+    policies.kyverno.io/minversion: 1.11.0
+    kyverno.io/kubernetes-version: "1.27-1.28"
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Legacy k8s.gcr.io container image registry will be frozen in early April 2023
+      k8s.gcr.io image registry will be frozen from the 3rd of April 2023.  
+      Images for Kubernetes 1.27 will not be available in the k8s.gcr.io image registry.
+      Please read our announcement for more details.
+      https://kubernetes.io/blog/2023/02/06/k8s-gcr-io-freeze-announcement/     
+spec:
+  validationFailureAction: Enforce
```

Comment:
Do we need it in the enforce mode?

Reply:
It is a strong requirement to point to `registry.k8s.io` from `k8s.gcr.io` ([Link](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/#tl-dr-what-you-need-to-know-about-this-change)). I think we can keep it in Enforce mode.